core/trino-main/src/test/java/io/trino/sql/query/QueryAssertions.java

```diff
@@ -430,10 +430,6 @@ public QueryAssert containsAll(@Language("SQL") String query)
             return this;
         }
 
-        /**
-         * @deprecated use {@code result().isEmpty()} instead.
-         */
-        @Deprecated
```

Comment:
re: " Undeprecate returnsEmptyResult() "

when using `isFullyPushedDown` the query runs twice with and without pushdown. Why is it useful to match what the result is? e.g. we don't use `matches` in push down tests for example.

Reply:
good question. when i introduced isFullyPushedDown i felt there are cases where it's valuable to assert results, as  away to witness that test query is reasonable.